api/controllers/game/concede.js

```diff
@@ -22,6 +22,10 @@ module.exports = async function (req, res) {
         victory,
       },
     });
+
+    // Inform all clients this game is over
+    sails.sockets.blast('gameFinished', { gameId: game.id });
```

Comment:
https://github.com/cuttle-cards/cuttle/issues/354

Reply:
As long as it's not majorly breaking anything (and it's already broken today), I think forking it off is a better idea since I much prefer more concise and pointed PRs.